docs/user/custom-resources/apirule/04-10-apirule-custom-resource.md

```diff
@@ -13,28 +13,28 @@ This table lists all parameters of APIRule CRD together with their descriptions:
+| **corsPolicy**                    | **NO**      | Allows configuring CORS headers sent with the response. If **corsPolicy** is not defined, the default values are applied. If **corsPolicy** is configured, only the CORS headers defined in the APIRule are sent with the response. For more information, see the [decision record](https://github.com/kyma-project/api-gateway/issues/752). |
+| **corsPolicy.allowHeaders**       | **NO**      | Specifies headers allowed with the **Access-Control-Allow-Headers** CORS header.                                                                                                                                                                                                                                                             |
+| **corsPolicy.allowMethods**       | **NO**      | Specifies methods allowed with the **Access-Control-Allow-Methods** CORS header.                                                                                                                                                                                                                                                             |
+| **corsPolicy.allowOrigins**       | **NO**      | Specifies origins allowed with the **Access-Control-Allow-Origins** CORS header.                                                                                                                                                                                                                                                             |
+| **corsPolicy.allowCredentials**   | **NO**      | Specifies whether credentials are allowed in the **Access-Control-Allow-Credentials** CORS header.                                                                                                                                                                                                                                           |
+| **corsPolicy.exposeHeaders**      | **NO**      | Specifies headers exposed with the **Access-Control-Expose-Headers** CORS header.                                                                                                                                                                                                                                                            |
+| **corsPolicy.maxAge**             | **NO**      | Specifies the maximum age of CORS policy cache. The value is provided in the **Access-Control-Max-Age** CORS header. The value type is "duration", for example 200s.                                                                                                                                                                         |
```

Comment:
```suggestion
| **corsPolicy.maxAge**             | **NO**      | Specifies the maximum age of CORS policy cache. The value is provided in the **Access-Control-Max-Age** CORS header. The value type is `duration`, for example, 200s.                                                                                                                                                                         |
```

Reply:
The format is `1h200s`, so it should stay as it is.
> ParseDuration parses a duration string. A duration string is a possibly signed sequence of decimal numbers, each with optional fraction and a unit suffix, such as "300ms", "-1.5h" or "2h45m". Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h", "d", "w", "y".

Reply:
So let's also use code font for this